src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const add = (x) => {
+    const currentSum = sum;
```

Comment:
A better place for this variable 'currentSum' is inside your `else` statement. Because you actually don't need this variable if x !== undefined. So let's put it there not to declare it every time (even if there's no need for this).